toolkit/tools/pkg/imagecustomizerlib/dracututils.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerlib
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+)
+
+func addDracutConfig(dracutConfigFile string, lines []string) error {
```

Comment:
How was the dracut logic extended?

Reply:
Previously we have only the function `addDracutModuleAndDriver` adding both module and driver, today we realized that a driver or a module can be added individually. We open this file and collect all the logics.